src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const totalPrice = price * days;
+  const threeDiscount = 20;
+  const sevenDiscount = 50;
+  let cost = 0;
+
+  if (days >= 7) {
+    cost = totalPrice - sevenDiscount;
+
+    return cost;
```

Comment:
you can return this calculation right away
```suggestion
    return totalPrice - sevenDiscount;
```